pkg/gather/service/analyze.go

```diff
@@ -84,6 +84,7 @@ func analyzeGatherBundle(bundleFile io.Reader) error {
 		check    func(analysis) bool
 		optional bool
 	}{
+		{name: "node-image-pull", check: checkReleaseImageDownload, optional: false},
```

Comment:
I was thinking about adding unit tests case for `node-image-pull` in:

https://github.com/openshift/installer/blob/88ba667efee96aaa4a181843262f535c78a3a3ec/pkg/gather/service/analyze_test.go#L48

But the service `release-image` and `node-image-pull` are handled the same way. Let's just rename test cases to `node-image-pull` instead to avoid dups + reflect the new "actually being used" service?